kedro-datasets/tests/snowflake/test_snowpark_dataset.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import os
+
+import pandas as pd
+import pytest
+from kedro.io import DataSetError
+
+try:
+    import snowflake.snowpark as sp
+
+    from kedro_datasets.snowflake import SnowparkDataSet as spds
+except ImportError:
+    pass  # this is only for test discovery to succeed on Python <> 3.8
+
+
+def get_connection():
+    account = os.getenv("SF_ACCOUNT")
+    warehouse = os.getenv("SF_WAREHOUSE")
```

Comment:
it will good to consider for example SNOWCLI compatible environment variables. 
```
  sf_user      = os.getenv("SNOW_USER")     or os.getenv("SNOWSQL_USER")
        sf_password  = os.getenv("SNOW_PASSWORD") or os.getenv("SNOWSQL_PWD")
        sf_account   = os.getenv("SNOW_ACCOUNT")  or os.getenv("SNOWSQL_ACCOUNT")
        sf_role      = os.getenv("SNOW_ROLE")     or os.getenv("SNOWSQL_ROLE")
        sf_warehouse = os.getenv("SNOW_WAREHOUSE") or os.getenv("SNOWSQL_WAREHOUSE")
        sf_database  = os.getenv("SNOW_DATABASE") or os.getenv("SNOWSQL_DATABASE")
        if sf_user is not None:
            self._options["user"]     = os.getenv("SNOW_USER") or os.getenv("SNOWSQL_USER")
        if sf_password is not None:
            self._options["password"] = sf_password
        if sf_account is not None:
            self._options["account"]  = sf_account
        if sf_role is not None:
            self._options["role"]     = sf_role
        if sf_warehouse is not None:
            self._options["warehouse"]= sf_warehouse
        if sf_database is not None:
            self._options["database"] = sf_database
            ```

Reply:
thanks for the suggestion, standardized according to the [SnowSQL docs](https://docs.snowflake.com/en/user-guide/snowsql-start.html#label-snowsql-connection-parameters)